Formula/l/llamafile.rb

```diff
@@ -0,0 +1,43 @@
+class Llamafile < Formula
+  desc "Distribute and run LLMs with a single file"
+  homepage "https://github.com/Mozilla-Ocho/llamafile"
+  url "https://github.com/Mozilla-Ocho/llamafile/archive/refs/tags/0.6.tar.gz"
+  sha256 "9ac8449268287c45fb015f7068b223be7336c6d7f8170f3adb9a370b65e8b9f0"
+  license "Apache-2.0"
+  head "https://github.com/Mozilla-Ocho/llamafile.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
Does the default livecheck not work?

Reply:
TIL! It does. Dropped this in block 0d537af93707e524985cd895deed31f9980847bf